### PR TITLE
Feat/summary

### PR DESCRIPTION
**Pull request by edwardbaeg**
Adds a summary component.

@ArthurHovanesian, please let me know what you think!

### CODE REVIEW

File: client/components/bookingButton.jsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+import styles from '../styles/bookingButton.css';
+
+const BookingButton = () => {
+  return (
+    <div>
+      <button className={styles.button} type="button">
```
Comment (ArthurHovanesian) [inline client/components/bookingButton.jsx:8]: Maybe space before and after the curly brackets for readability.

File: client/styles/bookingButton.css [hunk 2]
```diff
@@ -0,0 +1,19 @@
+.button {
+  background-color: #FF5A5F;
+  border: none;
+  border-radius: 4px;
+  color: white;
+  cursor: pointer;
+  font-size: 16px;
+  padding-top: 12px;
```
Comment (ArthurHovanesian) [inline client/styles/bookingButton.css]: Consider using shorthand syntax if top and bottom have same padding

**Review by ArthurHovanesian — APPROVED** [review]

File: client/components/bookingButton.jsx (hunk 1 above)
Comment (edwardbaeg) [reply client/components/bookingButton.jsx:8]: I believe this goes against the AirBnB style guide. I think that's expected for object destructuring

File: client/styles/bookingButton.css (hunk 2 above)
Comment (edwardbaeg) [reply client/styles/bookingButton.css]: Good catch!

**Pull request merged**